### PR TITLE
PMM-6230 Keep changes in Home Dashboard during the updates

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/969

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -420,16 +420,22 @@ def set_home_dashboard(api_key):
...
     r = requests.get('%s/api/dashboards/db/home-dashboard' % (HOST,), headers=grafana_headers(api_key))
-    print ' * "home" dashboard: %r %r' % (r.status_code, r.content)
+    print ' * PMM "home" dashboard: %r %r' % (r.status_code, r.content)
     if r.status_code != httplib.OK:
         # TODO sys.exit(-1)
         return
 
-    res = json.loads(r.content)
+    # Get parameters of current home dashboard
+    h = requests.get('%s/api/dashboards/home' % (HOST,), headers=grafana_headers(api_key))
```
Comment (askomorokhov) [inline import-dashboards.py:429]: Could you please give more verbose name for this response?

File: import-dashboards.py [hunk 2]
```diff
@@ -420,16 +420,22 @@ def set_home_dashboard(api_key):
...
 
-    data = json.dumps({'homeDashboardId': res['dashboard']['id']})
-    r = requests.put('%s/api/user/preferences' % (HOST,), data=data, headers=grafana_headers(api_key))
-    print ' * Preferences set: %r %r' % (r.status_code, r.content)
+    # Check if any dashboard has set as the home dashboard
+    if not 'redirectUri' in json.loads(h.content):
```
Comment (askomorokhov) [inline import-dashboards.py:432]: Is this the only way to detect if home dashboard was set?

Comment (askomorokhov) [inline import-dashboards.py:432]: if we not use result of json.loads(h.content), may be we can simply check is string contains substring?
Ex.: `if not 'redirectUri' in h.content:`

File: import-dashboards.py (hunk 1 above)
Comment (adivinho) [reply import-dashboards.py]: It has been renamed to homeDashboard

File: import-dashboards.py (hunk 2 above)
Comment (adivinho) [reply import-dashboards.py]: I can't find any other method.

Screenshot_20200720_130716

Comment (adivinho) [reply import-dashboards.py]: Yes, it can be simplified.

**Pull request merged**